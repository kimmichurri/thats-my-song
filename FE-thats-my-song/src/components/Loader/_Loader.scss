.loading-icon-wrapper {
  display: flex;
  justify-content: center;
}

.loading-icon {
  animation: spin 2s linear infinite;
  width: 125px;
  height: 125px;
}

@keyframes spin {
  0% { transform: rotate(0deg) }
  100% { transform: rotate(360deg)}
}
