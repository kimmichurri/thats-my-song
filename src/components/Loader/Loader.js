import React from 'react';
import loader from '../../assets/loader.png';

export default function Loader() {
  return (
    <div className='loading-icon-wrapper'>
      <img
        className='loading-icon'
        src={loader}
        alt='loading icon, the page is loading'
      />
    </div>
  )
}