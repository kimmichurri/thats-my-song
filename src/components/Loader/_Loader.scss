.loading-icon-wrapper {
  display: flex;
  justify-content: center;
}

.loading-icon {
  animation: spin 2s linear infinite;
  width: 275px;
  height: 275px;
}

@keyframes spin {
  0% { transform: rotate(0deg) }
  100% { transform: rotate(360deg)}
}
