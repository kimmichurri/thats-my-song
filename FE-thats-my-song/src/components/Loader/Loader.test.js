import Loader from './Loader';
import React from 'react';
import { shallow } from 'enzyme';

describe('Loader', () => {
  it('should match the snapshot', () => {
    const wrapper = shallow(<Loader />);
    expect(wrapper.debug()).toMatchSnapshot();
  });
});