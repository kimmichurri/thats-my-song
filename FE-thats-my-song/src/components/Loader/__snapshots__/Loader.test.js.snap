// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader should match the snapshot 1`] = `
"<div className=\\"loading-icon-wrapper\\">
  <img className=\\"loading-icon\\" src=\\"loader.png\\" alt=\\"loading icon, the page is loading\\" />
</div>"
`;
